src/kvdb/memory_database.go

```diff
@@ -71,7 +70,7 @@ func (w *MemDatabase) Get(key []byte) ([]byte, error) {
 	if entry, ok := w.db[string(key)]; ok {
 		return common.CopyBytes(entry), nil
 	}
-	return nil, errors.New("not found")
+	return nil, nil
```

Comment:
Why are you removing the error?

Reply:
Get() should return nil if item not found. It is not error. See the same behavior at BadgerDatabase.Get().